### PR TITLE
project-template 0.9.0

### PR DESCRIPTION
**Pull request by mt-edwards**
- removed .gitkeep files
- moved models/ directory from data/ directory
- removed config.yaml (not ready for release)
- updated README.md file
- moved venv and git post hooks to makefile
- updated .gitignore

### CODE REVIEW

File: {{cookiecutter.project_directory_name}}/Makefile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#################################################################################
+# COMMANDS                                                                      #
+#################################################################################
+
+venv:
```
Comment (m-misiura) [inline {{cookiecutter.project_directory_name}}/Makefile:5]: should there be an R-equivalent of virtual envs ?

**Review by m-misiura — CHANGES_REQUESTED** [review]
I've reviewed all the files:

- made minor typo fix in README.md: visual, not virtual studio code
- created the directory and used `make` to set-up the python venv, this works fine

There are two outstanding comments that I would like to clarify before approving these changes:

1. in the setup part of README.md, should we have a description of how to e.g. activate these venvs?
2. I remember you showing me how to do venvs for R, but this is not part of the Makefile - is this correct ?

File: README.md [hunk 2]
```diff
@@ -18,75 +18,70 @@ You will be prompted for eleven inputs:
...
-## Guide
+To setup `venv`:
 
-### Clean Data
-
-![](figures/clean.drawio.svg)
+```
+make venv
+make requirements
+```
```
[the remark this answers is not in the document]
Comment (mt-edwards) [reply README.md]: This will be addressed in the next pull request.

File: {{cookiecutter.project_directory_name}}/Makefile (hunk 1 above)
Comment (mt-edwards) [reply {{cookiecutter.project_directory_name}}/Makefile]: This will be addressed in the next pull request.

**Pull request closed without merging**